reporting-engine-worker/src/main/java/org/ihtsdo/termserver/scripting/reports/release/PackageComparisonReport.java

```diff
@@ -61,9 +85,39 @@ public Job getJob() {
 				.withExpectedDuration(40)
 				.build();
 	}
+
+	@Override
+	protected void loadProjectSnapshot(boolean fsnOnly) throws TermServerScriptException, InterruptedException, IOException {
+		//getArchiveManager(true).loadSnapshot(fsnOnly);
+		//Reset the report name to null here as it will have been set by the Snapshot Generator
+		//setReportName(null);
+	}
 	
 	public void runJob() throws TermServerScriptException {
 		//Kick off scripts here
+		boolean isWindows = System.getProperty("os.name").toLowerCase().startsWith("windows");
+
+		ProcessBuilder builder = new ProcessBuilder();
+		if (isWindows) {
+			builder.command("cmd.exe", "/c", "dir");
+		} else {
+			String command = "./" + scriptName + " " + String.join(" ", scriptArguments);
+			builder.command("sh", "-c", command);
+		}
+		builder.directory(new File("reporting-engine-worker/scripts"));//System.getProperty("user.home")));
+
+		try {
+			Process process = builder.start();
+
+			try (BufferedReader reader = new BufferedReader(new InputStreamReader(process.getInputStream()))) {
+				reader.lines().forEach(line -> System.out.println(line));
+			}
+
+			int exitVal = process.waitFor();
```

Comment:
I'm worried that we could end up with a locked up reporting engine worker here.   Is there an option for a maximum execution time timeout?    Again, not something to hold up the PR for.